docker-compose.yml

```diff
@@ -35,19 +101,25 @@ services:
 
   listener:
     build:
-      context: $PWD
+      context: $PWD/docker-files/listener
     image: candig/listener-service:${LISTENER_VERSION:-latest}
     init: true
-    #volumes:
-      #- $PWD:/srv
-    ports:
-      - "${LISTENER_PORT:-8080}:8080"
+    volumes:
+      - $PWD:/srv
+    #ports:
```

Comment:
Port still needs to be available, we can simply change the `LISTENER_PORT` to something else without having to change the flask server code

Reply:
Great idea! Solved in a49a74f